mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'two-third-third.html' %}
+
+{% load i18n static %}
+
+{% block twothird %}
+    <h2><span class="theme read">{{ document.get_type_display }}</span> {{ document }}</h2>
+    {% if document.type == document.IMAGE %}
+        <img src="{{ document.original.url }}" />
+    {% elif document.type == document.VIDEO %}
+        <video controls width="100%">
```

Comment:
Yep, one good point of the project: we now which browsers will connect to the server, only FF on desktop, and Webkit default on the tablets, so only modern ones :)